src/consumer/config/component-config.ts

```diff
@@ -347,10 +346,10 @@ export default class ComponentConfig extends AbstractConfig {
       });
     } catch (err) {
       // @todo: once we have a way to indicate this to the user, remove the next two lines
-      logger.console('runOnLoadEvent failed loading a load event');
-      logger.console(err);
-      logger.warn('extension on load event throw an error');
-      logger.warn(err);
+      // logger.console('runOnLoadEvent failed loading a load event');
```

Comment:
`logger.console` writes to the console. `logger.warn` writes to the log file. so, the last two lines we need for sure. 
for now, because we don't have a better mechanism of showing extensions error, I'd leave also the console logging.

Reply:
As discussed, @GiladShoham will take care.

Reply:
Already solved in harmony main. Please merge

Reply:
Great!! Can I now access components by variants pattern now?